config/arke/strategy.yaml

```diff
@@ -0,0 +1,15 @@
+strategy:
```

Comment:
I'm not sure you need to commit this

Reply:
yeah, me too, but I saw config files for other services laying down in `/config` by default like this, so I just kept the convention

Reply:
config:render task does not overwrite by default, it will be simplier to delete rendered file, then I'll get my own after I render config